src/episodeCode.tsx

```diff
@@ -0,0 +1,11 @@
+export function episodeCode(season: number, number: number): string {
+  let episodeCode = "";
+  if (number < 10) {
+    episodeCode = `E0${number}`;
+  } else {
+    episodeCode = `E${number}`;
+  }
+  const seasonCode: string = season < 10 ? `S0${season}` : `S${season}`;
+
+  return seasonCode + episodeCode;
+}
```

Comment:
you have an if statement & a ternary operator statement here - might be clearer to use one or the other so they are both the same